### DISCUSSION TITLE
Update to 1.12.5

### DISCUSSION THREAD

**Discussion by mariusfranzen** [net-lisias-ksp/KSP-Recall#58] [Support] [answered]
Idk maybe the current version works for 1.12.5, but it's listed as incompatible on CKAN.

If it's an issue on CKAN's end, is it possible to mark it as compatible there?

**Comment by Lisias** [comment]
Hi!

I'm testing the thing on this weekend, and as far as I now, you can mark it as compatible while I do not update KSP-Recall.

**Comment by Lisias** [comment, marked as answer]
I will update Recall and TweakScale with new bug fixes and avoiding pesking 1.12.5 users somewhere between today and tomorrow!

Cheers!
